pyqtgraph/Qt.py

```diff
@@ -317,7 +317,7 @@ def isQObjectAlive(obj):
     loadUiType = uic.loadUiType
 
     QtCore.Signal = QtCore.pyqtSignal
-    
+    QtCore.Slot   = QtCore.pyqtSlot
```

Comment:
I'm not convinced we want/need the slot decorator; it's been discussed that there might be some performance benefits to using it; but I think that's somewhat arguable.  I would propose for the purposes of this PR to remove it.

Reply:
Fair enough. It seems that this works without the slot decorator. --> done.